src/js/index.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+// test
+
+const $total = document.querySelector("#total");
+const $digits = document.querySelector(".digits");
+const $operations = document.querySelector(".operations");
+const $modifier = document.querySelector(".modifier");
+
+let isOperationClicked = false;
+let isOperator2In = false; // 두번째 인자가 들어왔는지 체크
+let operator1 = 0;
+let operator2 = 0;
```

Comment:
```suggestion
let isRightOperandIn = false; // 두번째 인자가 들어왔는지 체크
let rightOperand = 0;
let leftOperand = 0;
```

operator는 연산자 operand는 피연산자 이름이기에 좀더 정확히 적어주는게 좋을것 같아요
변수명 1,2는 조금 모호한 네이밍인거 같고 조건에서 두개의 피연산자를 계산하니 leftOperand, rightOperand 이렇게 네이밍하는게 더 괜찮지 않나 생각합니다:)